### DISCUSSION TITLE
멋진 업데이트!

### DISCUSSION THREAD

**Discussion by PulpExposed** [pronist/hELLO#252] [Ideas]
헬로 스킨은 사진 포트폴리오로 너무나 훌륭합니다.
몇가지 새스킨 적용 후 향후 업데이트에 반영할 수 있는지 문의드립니다.

1. 사진을 열람하는데 가로 비율 사진은 매우 자연스럽게 네비게이션이 가능하나, 세로 사진은 스크롤해서 볼 수 있을 뿐 전체 조망이 어렵습니다. 혹시 관련한 업데이트를 기대해도 될까요.
2. 완전 영문 또는 한글을 최소화한 픽토그램 스킨으로 발전할 수 있을까요. ex) 글 목록보기에서 "🤍 공감" → "🤍" 으로 변경.

이번에 이전보기, 다음 보기를 적용하며 너무나 행복한 시간을 보내고 있습니다. 
지금으로도 충분히 기능을 사용하고 있으며, 다음 업데이트가 어떤 방향이 될지 즐거운 상상 중입니다.
날이 더운데 늘 건강하시고 행복한 하루되셨으면 합니다.
감사합니다.

**Comment by pronist** [comment]
@PulpExposed 스킨을 좋아해주셔서 정말 감사합니다!

1. 세로 사진의 경우 세로 비율 사진을 표현할 수 있는 포트폴리오 형태의 홈커버의 개발 및 상품화 예정에 있습니다. 다만 글 내부에서는 현재도 적용되어 있는, 이미지를 클릭했을 때 나타나는 갤러리 박스(라이트 박스) 이외에 세로 사진을 전체적으로 볼 수 있는 요소를 추가하기에 아이디어가 떠오르지 않는 상황입니다.
2. 공감이나 공유하기와 같은 글 아래의 부분은 스킨이 아닌 티스토리에서 직접 주입하는 부분이라 스킨 개발자가 건드리기에도 제약이 따르고, 사용자가 스킨 편집으로 커스텀하는 것도 사실상 불가능한 부분입니다. 이 친구는 단순하게 On/Off 형태로 디스플레이 여부만 조절하는 것으로 나아가지 않을까 싶습니다. 그 외에 스킨에 적용된 한글 문구를 영문으로 나타낼 수 있는 다국어 기능의 경우 구현을 고려 중입니다.

아이디어 및 개선사항 제시 정말 감사드립니다. :))
